docs/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
It's possible to generate this page from mld file but I couldn't figure it out how to make it work well. The problem is that I always got some navigation elements at the top that I didn't want.